src/core/viewports/orbit-viewport.js

```diff
@@ -43,30 +45,35 @@ export default class OrbitViewport extends Viewport {
     translationY = 0, // in pixels
     zoom = 1
   }) {
-    const eye = vec3_add([], lookAt, [0, 0, distance]);
-    vec3_rotateX(eye, eye, lookAt, rotationX / 180 * Math.PI);
-    vec3_rotateY(eye, eye, lookAt, rotationY / 180 * Math.PI);
+    const rotationMatrix = mat4_rotateX([], createMat4(), -pitchAngle / 180 * Math.PI);
```

Comment:
Earlier, we were rotating 'eye' about 'lookAt', but now we move these rotations into world (as part of transformMatrix), but now we are rotating about Origin, if `lookAt` is not origin wouldn't this be completely different rotations?

Reply:
good catch, will fix it. Thanks!

Reply:
Done.